### PR TITLE
DDF-2824 Fixed some typos in IngestCommand and some configurations

### PR DESCRIPTION
**Pull request by emmberk**
#### What does this PR do?
- Fixes some typos in `IngestCommand`
- Fixes some typos in the `Expansion Service` metatypes and docs
- Adds a note about relative file paths to the `Schematron File Names` field in the `Schematron Validators` configuration
#### Who is reviewing it? 
@kcover @peterhuffer
#### Select relevant component teams: 
@codice/docs 
#### Choose 2 committers to review/merge the PR. 
@clockard
@ricklarsen
#### How should this be tested?
Full build. Check that tooltips and docs are formatted correctly.
#### What are the relevant tickets?
[DDF-2824](https://codice.atlassian.net/browse/DDF-2824)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java [hunk 1]
```diff
@@ -227,7 +227,7 @@
     aliases = {},
     multiValued = false,
     description =
-        "Ingest a zip file that contains metacards and content using the default transformer.  The specified zip must be signed externally using DDF certificates."
+        "Ingest a zip file that contains metacards and content using the default transformer. The specified zip must be signed externally using DDF certificates."
```
Comment (ricklarsen) [inline catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java]: 👍

File: platform/security/expansion/security-expansion-metacardattr-map/src/main/resources/OSGI-INF/metatype/metatype.xml [hunk 2]
```diff
@@ -18,14 +18,13 @@
     <OCD description="Expansion Service for Metacard Attributes"
          name="Security Metacard Attribute Mapping"
          id="ddf.security.metacard.attribute.mapping">
-        <AD description="Attribute separator - used to separate attributes in replacement strings - defaults to space (specified by no value)."
-            name="Attribute Separator" id="attributeSeparator" required="false"
-            type="String" default=" "/>
+        <AD description="Used to separate attributes in replacement strings. Defaults to space (specified by no value)."
```
Comment (clockard) [inline platform/security/expansion/security-expansion-metacardattr-map/src/main/resources/OSGI-INF/metatype/metatype.xml]: Is there a doc table that should be updated as well?

File: platform/security/expansion/security-expansion-userattr-map/src/main/resources/OSGI-INF/metatype/metatype.xml [hunk 3]
```diff
@@ -18,14 +18,13 @@
     <OCD description="Expansion Service for User Attributes"
          name="Security User Attribute Mapping"
          id="ddf.security.user.attribute.mapping">
-        <AD description="Attribute separator - used to separate attributes in replacement strings - defaults to space (specified by no value)."
-            name="Attribute Separator" id="attributeSeparator" required="false"
-            type="String" default=" "/>
+        <AD description="Used to separate attributes in replacement strings. Defaults to space (specified by no value)."
```
Comment (clockard) [inline platform/security/expansion/security-expansion-userattr-map/src/main/resources/OSGI-INF/metatype/metatype.xml]: Same comment as above.

File: platform/security/expansion/security-expansion-metacardattr-map/src/main/resources/OSGI-INF/metatype/metatype.xml (hunk 2 above)
Comment (emmberk) [reply platform/security/expansion/security-expansion-metacardattr-map/src/main/resources/OSGI-INF/metatype/metatype.xml]: I couldn't find one.

**Comment by emmberk** [comment]
build now

File: catalog/schematron/catalog-schematron-plugin/src/main/resources/OSGI-INF/metatype/metatype.xml [hunk 4]
```diff
@@ -14,23 +14,23 @@
...
 
-    <AD name="Root Namepsace" id="namespace" required="true" type="String"
-        description="The root namespace of the XML" />
+        <AD name="Root Namespace" id="namespace" type="String"
+            description="The root namespace of the XML"/>
 
-    <AD name="Schematron Files" id="schematronFileNames" required="true" type="String" cardinality="100"
-        description="Schematron files (*.sch) to be validated against" />
+        <AD name="Schematron File Names" id="schematronFileNames" type="String" cardinality="100"
+            description="Names of schematron files (*.sch) against which to validate metadata ingested into the Catalog. Absolute paths or relative paths may be specified. Relative paths are assumed to be relative to `${home_directory}/schematron`."/>
```
Comment (peterhuffer) [inline catalog/schematron/catalog-schematron-plugin/src/main/resources/OSGI-INF/metatype/metatype.xml]: ❓ Just curious where this `home_directory` property is being resolved from? Why not `ddf.home`?

Comment (emmberk) [reply catalog/schematron/catalog-schematron-plugin/src/main/resources/OSGI-INF/metatype/metatype.xml]: https://github.com/codice/ddf/blob/d935a855ea9676af76ec0cce32e272f249bf43bd/pom.xml#L207
It gets resolved to `<DDF_HOME>`, which means that it won't get filled in with the full path that to which ${ddf.home} resolves.

File: catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java [hunk 5]
```diff
@@ -197,7 +197,7 @@
     aliases = {"-d", "-f", "Ingest Failure Directory"},
     multiValued = false,
     description =
-        "The directory to put files that failed to ingest.  Using this option will force a batch size of 1."
+        "The directory to put file(s) that failed to ingest. Using this option will force a batch size of 1."
```
Comment (emmberk) [inline catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java]: `The directory in which to put ...`

**Comment by emmberk** [comment]
```
16:36:40 [ERROR] Failed to execute goal org.owasp:dependency-check-maven:1.4.5:check (default) on project catalog-core-commands:
16:36:40 [ERROR] 
16:36:40 [ERROR] One or more dependencies were identified with vulnerabilities that have a CVSS score greater then '7.0':
16:36:40 [ERROR] 
16:36:40 [ERROR] jackson-databind-2.5.4.jar: CVE-2017-15095
16:36:40 [ERROR] htrace-core-3.2.0-incubating.jar/META-INF/maven/com.fasterxml.jackson.core/jackson-databind/pom.xml: CVE-2017-15095
16:36:40 [ERROR] ehcache-2.10.4.jar/rest-management-private-classpath/META-INF/maven/com.fasterxml.jackson.core/jackson-databind/pom.xml: CVE-2017-15095
16:36:40 [ERROR] 
16:36:40 [ERROR] See the dependency-check report for more details.
16:36:40 [ERROR] -> [Help 1]
16:36:40 [ERROR] 
16:36:40 [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
16:36:40 [ERROR] Re-run Maven using the -X switch to enable full debug logging.
16:36:40 [ERROR] 
16:36:40 [ERROR] For more information about the errors and possible solutions, please read the following articles:
16:36:40 [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
16:36:40 [ERROR] 
16:36:40 [ERROR] After correcting the problems, you can resume the build with the command
16:36:40 [ERROR]   mvn <goals> -rf :catalog-core-commands
```

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
`Failed to execute goal org.jacoco:jacoco-maven-plugin:0.8.0:check (default-check) on project security-core-api: Coverage checks have not been met.`

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
`Failed to execute goal org.jacoco:jacoco-maven-plugin:0.8.0:check (default-check) on project security-core-api: Coverage checks have not been met.`

**Comment by emmberk** [comment]
build now

**Pull request merged**